### PR TITLE
fix(bridge): Truncate evaluation score

### LINKED ISSUE #6890: Rounding evaluation score causes confusion on evaluation board
### Problem

When having an evaluation score of 89.69... it is still below the warning threshold of, e.g. 90, but due to rounding this value it looks like being exactly 90 and this violates the expression 90 < 90 shown below:

issue

### Proposed fix

* Round the value to two decimals. Given the above example, this would be `89.68`

### PR DESCRIPTION
**Pull request by ermin-muratovic**
## This PR
- is truncating the evaluation score correctly

### Related Issues
Fixes #6890 

image

### CODE REVIEW

**Review by laneli — CHANGES_REQUESTED** [review]
Can you please add Unit Tests for any other case that is not 1.99999, to be sure that this also is still working. Maybe for one for different decimals and one for a number without decimals?

File: bridge/client/app/_pipes/truncate-number.pipe.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { TruncateNumberPipe } from './truncate-number';
+
+describe('TruncateNumberPipe', () => {
```
Comment (Kirdock) [inline bridge/client/app/_pipes/truncate-number.pipe.spec.ts:3]: Maybe also add a test for negative numbers and a case to truncate a number to 3 or 1 decimals (a parameter that isn't 2)

**Review by Kirdock — APPROVED** [review]

**Review by laneli — APPROVED** [review]
